src/app/services/auth/auth.service.ts

```diff
@@ -65,6 +68,16 @@ export class AuthService {
     }));
   }
 
+  setupPeriodicTokenGeneration(): void {
+    if (!this.generateTokenSubscription || this.generateTokenSubscription.closed) {
+      this.generateTokenSubscription = timer(0, 290 * 1000).pipe(
```

Comment:
Let's extract 300 to a variable to explain that this is a default middleware token duration.